### DISCUSSION TITLE
Error: Empty template: Front Page

### DISCUSSION THREAD

**Discussion by solidcodes** [WordPress/gutenberg#43240] [Developer Experience]
I'm using Gutenberg and Twentytwentytwo theme.
The frontpage is set to static page "home"
Whenever I browse the frontpage it displays a white page.
With an error message "Empty template: Front Page"

Is this Gutenberg bug or Twentytwentytwo bug?

How do I fix this?
Thank you in advance.

**Comment by solidcodes** [comment]
I tried removing TWentytwentytwo theme and temporarily using Kadence Theme.
and it seems the error bug is gone.
So this is a problem with Twentytwentytwo theme.

Now I need to findout which file is causing this problem.

**Comment by talldan** [comment]
@solidcodes I had a go at setting my front page to a page like you, and it seemed to work ok. Which WordPress version are you on?

If you go to Appearance > Editor > Click the W icon in the top left corner > Click 'Templates' from the sidebar that opens. Do you have a 'Page' template in the list of templates? This is how mine looks by default:
![Screen Shot 2022-08-17 at 2 51 31 pm](https://user-images.githubusercontent.com/677833/185053750-45990e3f-a35a-4c49-8a70-b145b8fc5755.png)
